docs/gen-docs/kyma_deploy.md

```diff
@@ -0,0 +1,102 @@
+
+  Deploy Kyma from specific source:
+    - Deploy from a specific version, such as 1.19.1:
+		kyma deploy --source=1.19.1
+    - Build Kyma from local sources and deploy on remote cluster:
+		kyma deploy --source=local
+
+  Deploy Kyma with only specific components:
+    You need to pass a path to a YAML file containing desired components. An example YAML file would contain:
+		prerequisites:
+		- name: "cluster-essentials"
+		- name: "istio"
+		  namespace: "istio-system"
+		components:
+		- name: "testing"
+		- name: "xip-patch"
+		- name: "istio-kyma-patch"
+		- name: "dex"
+    Then run:
+		kyma deploy --components {COMPONENTS_FILE_PATH}
+
+  Change Kyma settings:
+    To change your Kyma configuration, use the deploy command and deploy the same Kyma version that you're currently using,
+    just with different settings.
+    - Using a settings-file:
+		kyma deploy --values-file {VALUES_FILE_PATH}
+    - Using specific values instead of file:
+		kyma deploy --value ory.hydra.deployment.resources.limits.cpu=153m \
+		--value ory.hydra.deployment.resources.requests.cpu=53m
+
+Debugging:
+  The commands support troubleshooting in several ways, for example:
+  - To get a detailed view of the installation process, use the --verbose flag.
+  - To tweak the values on a component level, use deploy --components:
+    Pass a components list that includes only the components you want to test
+    and try out the settings that work for your installation.
+  - To understand which component failed during deployment, deactivate the default atomic deployment:
+		--atomic=false
+    With atomic deployment active, any component that hasn't been installed successfully is rolled back,
+    which may make it hard to find out what went wrong. By disabling the flag, the failed components are not rolled back.
+	
+
+```bash
+kyma deploy [flags]
+```
+
+## Flags
+
+```bash
+  -a, --atomic                       Set --atomic=true to use atomic deployment, which rolls back any component that could not be installed successfully.
+      --component strings            Provide one or more components to deploy (e.g. --component componentName@namespace)
+  -c, --components-file string       Path to the components file (default "$HOME/.kyma/sources/installation/resources/components.yaml" or ".kyma-sources/installation/resources/components.yaml")
+      --concurrency int              Number of parallel processes (default 4)
+  -d, --domain string                Custom domain used for installation
+  -p, --profile string               Kyma deployment profile. If not specified, Kyma uses its default configuration. The supported profiles are: "evaluation", "production".
+  -r, --reuse-values                 Set --reuse-values=true to reuse the helm values during component installation
+  -s, --source string                Installation source:
+                                     	- Deploy a specific release, for example: "kyma deploy --source=1.17.1"
```

Comment:
as alpha commands will only be stable with kyma 2.0, I'd suggest to replace the example version number :)